src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -4780,25 +4781,16 @@ namespace ZoneTempPredictorCorrector {
 		H2OHtOfVap = PsyHgAirFnWTdb( ZoneAirHumRat( ZoneNum ), ZT( ZoneNum ) );
 
 		// Check for the flow and NO flow condition
-		if ( ZoneMassFlowRate > 0.0 ) {
-			B = ( LatentGain / H2OHtOfVap ) + ( ( OAMFL( ZoneNum ) + VAMFL( ZoneNum ) + CTMFL( ZoneNum ) ) * OutHumRat ) + EAMFLxHumRat( ZoneNum ) + ( MoistureMassFlowRate ) + SumHmARaW( ZoneNum ) + MixingMassFlowXHumRat( ZoneNum ) + MDotOA( ZoneNum ) * OutHumRat;
-			A = ZoneMassFlowRate + OAMFL( ZoneNum ) + VAMFL( ZoneNum ) + EAMFL( ZoneNum ) + CTMFL( ZoneNum ) + SumHmARa( ZoneNum ) + MixingMassFlowZone( ZoneNum ) + MDotOA( ZoneNum );
-			if ( SimulateAirflowNetwork == AirflowNetworkControlMultizone || SimulateAirflowNetwork == AirflowNetworkControlMultiADS || ( SimulateAirflowNetwork == AirflowNetworkControlSimpleADS && AirflowNetworkFanActivated ) ) {
-				// Multizone airflow calculated in AirflowNetwork
-				B = ( LatentGain / H2OHtOfVap ) + ( AirflowNetworkExchangeData( ZoneNum ).SumMHrW + AirflowNetworkExchangeData( ZoneNum ).SumMMHrW ) + ( MoistureMassFlowRate ) + SumHmARaW( ZoneNum );
-				A = ZoneMassFlowRate + AirflowNetworkExchangeData( ZoneNum ).SumMHr + AirflowNetworkExchangeData( ZoneNum ).SumMMHr + SumHmARa( ZoneNum );
-			}
-			C = RhoAir * Zone( ZoneNum ).Volume * Zone( ZoneNum ).ZoneVolCapMultpMoist / SysTimeStepInSeconds;
-		} else if ( ZoneMassFlowRate <= 0.0 ) {
-			B = ( LatentGain / H2OHtOfVap ) + ( ( OAMFL( ZoneNum ) + VAMFL( ZoneNum ) + CTMFL( ZoneNum ) + ExhMassFlowRate ) * OutHumRat ) + EAMFLxHumRat( ZoneNum ) + SumHmARaW( ZoneNum ) + MixingMassFlowXHumRat( ZoneNum );
-			A = OAMFL( ZoneNum ) + VAMFL( ZoneNum ) + EAMFL( ZoneNum ) + CTMFL( ZoneNum ) + ExhMassFlowRate + SumHmARa( ZoneNum ) + MixingMassFlowZone( ZoneNum );
-			if ( SimulateAirflowNetwork == AirflowNetworkControlMultizone || SimulateAirflowNetwork == AirflowNetworkControlMultiADS || ( SimulateAirflowNetwork == AirflowNetworkControlSimpleADS && AirflowNetworkFanActivated ) ) {
-				// Multizone airflow calculated in AirflowNetwork
-				B = ( LatentGain / H2OHtOfVap ) + SumHmARaW( ZoneNum ) + AirflowNetworkExchangeData( ZoneNum ).SumMHrW + AirflowNetworkExchangeData( ZoneNum ).SumMMHrW;
-				A = AirflowNetworkExchangeData( ZoneNum ).SumMHr + AirflowNetworkExchangeData( ZoneNum ).SumMMHr + SumHmARa( ZoneNum );
-			}
-			C = RhoAir * Zone( ZoneNum ).Volume * Zone( ZoneNum ).ZoneVolCapMultpMoist / SysTimeStepInSeconds;
+		ZoneOABal = MDotOA( ZoneNum );
```

Comment:
What is `MDotOA`?  I only see it used in the moisture prediction/correction, but I don't find it in the temperature calculations.

Reply:
@mjwitte MDotOA is defined as Airbalance MASS FLOW rate in DataHeatBalFanSys, The equivalent item of MDotOA in the temperature calculation is MDotCPOA defined as Airbalance MASS FLOW * AIR SPECIFIC HEAT.

In order to avoid large differences in an example file, the following two lines of correction are used:

		ZoneOABal = MDotOA( ZoneNum );
		if ( ZoneMassFlowRate <= 0.0 ) ZoneOABal = 0.0;
However, the temperature calculation uses MDotCPOA as is without modification. Therefore, in order to be consistent with energy calculation, the correct may be removed.

Reply:
@lgu1234 I still don't understand what "Airbalance mass flow rate" is.  Where does it come from?

Reply:
@mjwitte Let me dig deeper to find out "MDotOA".

Reply:
@mjwitte "MDotOA" is calculated with Air Balance Method = Quadrature in the ZoneAirBalance:OutdoorAir. It should be independent of ZoneMassFlowRate and calculated every time step. Therefore, the code in temperature calculation is correct, while the following two lines should be removed in moisture calculation, even though large differences are observed in SingleFamilyHouse_TwoSpeed_ZoneAirBalance.idf:

        ZoneOABal = MDotOA( ZoneNum );
	if ( ZoneMassFlowRate <= 0.0 ) ZoneOABal = 0.0;

Reply:
@lgu1234 Thanks.  I forgot about that feature.  I would be good to expand the comment for MDotOA where it's declared.  Please go ahead and make this correction and remove the unused variables than I'll give it one final review.